src/Hal/Application/Config/Configuration.php

```diff
@@ -55,6 +55,11 @@ class Configuration implements ConfigurationInterface
     private $ignoreErrors = false;
 
     /**
+     * @var array
+     */
+    private $extensions = array();
```

Comment:
Within the constructor you set `$this->extensions = new ExtensionsConfiguration();`